packages/jest-runtime/src/helpers.ts

```diff
@@ -10,6 +10,25 @@ import slash = require('slash');
 import glob = require('glob');
 import type {Config} from '@jest/types';
 
+const OUTSIDE_JEST_VM_PROTOCOL = 'jest-main:';
+// String manipulation is easier here, fileURLToPath is only in newer Nodes,
+// plus setting non-standard protocols on URL objects is difficult.
```

Comment:
```js
const url = new URL('http://example.com');

url.protocol = 'jest-is-da-bomb';

console.log(url.href) // jest-is-da-bomb://example.com/
```

Reply:
In the browser, yes. But not on any Node version I've tried. https://nodejs.org/api/url.html#url_special_schemes

Reply:
🙁 is chrome not spec compliant then?

doing `.replace(/^file/, 'jest-is-best')` isn't too bad, tho

Reply:
I would guess this is not a spec thing, Node probably came up with this. Or maybe the spec allows runtimes to decide on this but leaves it up to them which protocols, if any, are considered special. But I haven't checked in the spec